commons-geometry-core/src/test/java/org/apache/commons/geometry/core/GeometryTestUtils.java

```diff
@@ -68,15 +68,15 @@ public static void assertThrows(final Runnable r, final Class<?> exceptionType)
     public static void assertThrows(final Runnable r, final Class<?> exceptionType, final String message) {
         try {
             r.run();
-            Assert.fail("Operation should have thrown an exception");
+            Assertions.fail("Operation should have thrown an exception");
```

Comment:
Could this entire method be replaced with the JUnit assertThrows?

Reply:
Hi @darkma773r , 
I completely agree with you. but, what do you think to do this in a separate jira?  as a upgrade?

Reply:
A separate task sounds good.